README.md

```diff
@@ -23,7 +23,7 @@ Here are some resources Coder Quad recommends to prepare for OA's and technical
 | [Sofi](https://www.sofi.com/careers/job/?gh_jid=5110537003) | Seattle, San Francisco, Murrah | Software Development Engineer, New Grad |
 | [Egen Solutions](https://jobs.lever.co/egensolutions/b8241d47-ab6c-456c-823d-1a6bd59e402f) | Naperville, IL | Software Engineer, University Graduates, 2023 |
 | [Amazon](https://www.amazon.jobs/en/jobs/2080728/software-development-engineer-2023-us) | US | New Grad SDE |
-| [Sunshine](https://jobs.lever.co/sunshine/a576f278-9748-49ab-b0b5-1104ef8880c4/apply) | Palo Alto, CA | SWE New Grad 2023 | 
+| [Sunshine](https://jobs.lever.co/sunshine) | Palo Alto, CA | SWE New Grad 2023 | 
```

Comment:
```suggestion
| [~~Sunshine~~](https://jobs.lever.co/sunshine/a576f278-9748-49ab-b0b5-1104ef8880c4/apply) | Palo Alto, CA | SWE New Grad 2023 | 
```
The reason the link is no longer reachable, is because the job posting has closed. So instead of replacing the link to the application with the careers page, just cross out Sunshine.